packages/app/src/specs/banners/ConnectProjectBanner.vue

```diff
@@ -55,19 +56,27 @@ query ConnectProjectBanner {
 }
 `
 
-withDefaults(defineProps<{
+const props = withDefaults(defineProps<{
   modelValue: boolean
   hasBannerBeenShown: boolean
+  bodyCopyOptions?: CohortOption[]
 }>(), {
   modelValue: false,
   hasBannerBeenShown: true,
+  bodyCopyOptions: () => {
```

Comment:
Is this prop only used for testing purposes? Adding a comment would be useful here if so, I see this pattern on a few components (I assume it's to remove the random choice between A or B cohort)